toolsWebsite/tools/models.py

```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
-# Create your models here.
+class ToolCategory(models.Model):
+    type = models.CharField(max_length=30)
+    count = models.IntegerField(default=0)
+    unavailable = models.IntegerField(default=0)
```

Comment:
We should add a price field here as well, so we can say how much each tool will cost.

Reply:
Finished that up.